### PR TITLE
feat(check): Allow to only check trees+packs for given snapshots

### PR DESCRIPTION
**Pull request by aawsome**
see #251

### CODE REVIEW

**Comment by simonsan** [comment]
Unblocked after merging #1282

File: src/commands/check.rs [hunk 1]
```diff
@@ -27,7 +31,16 @@ impl CheckCmd {
     fn inner_run(&self) -> Result<()> {
         let config = RUSTIC_APP.config();
         let repo = open_repository(&config.repository)?;
-        repo.check(self.opts)?;
+
+        let groups = repo.get_snapshot_group(&self.ids, SnapshotGroupCriterion::new(), |sn| {
```
Comment (aawsome) [inline src/commands/check.rs:35]: I think there is also get_matching_snapshots which is better here?!?

Comment (simonsan) [reply src/commands/check.rs]: Pushed some changes, did you mean it like that?

Comment (aawsome) [reply src/commands/check.rs]: Uh, no. I realize that we have `self.ids` - this must be why I used `get_snapshot_group`... I think then it is best to let it as it was... sorry for leading into the wrong direction..

Comment (simonsan) [reply src/commands/check.rs]: No problem ;-)

**Review by simonsan — APPROVED** [review]

**Pull request merged**